session/handler.go

```diff
@@ -178,6 +180,15 @@ func (h *Handler) whoami(w http.ResponseWriter, r *http.Request, ps httprouter.P
 		return
 	}
 
+	// Refresh session if param was true
+	refresh := r.URL.Query().Get("refresh")
```

Comment:
This with conjunction with CSRF protection disabled for this endpoint is a risky endeavour.  I know it was discussed in https://github.com/ory/kratos/issues/615 but I'm not sure what the conclusion was. For our use case (mobile apps) having two calls or refresh token seems like a reasonable compromise.

Reply:
this is enabled only in the config. This was requested by the community, but is disabled by default